mirdata/datasets/phenicx_anechoic.py

```diff
@@ -0,0 +1,422 @@
+import jams
+import librosa
+import numpy as np
+from typing import BinaryIO, Optional, TextIO, Tuple, cast
+
+from mirdata import download_utils
+from mirdata import jams_utils
+from mirdata import core
+from mirdata import annotations
+from mirdata import io
+
+
+BIBTEX = """
+@article{miron2016score,
+  title={Score-informed source separation for multichannel orchestral recordings},
+  author={Miron, Marius and Carabias-Orti, Julio J and Bosch, Juan J and G{\'o}mez, Emilia and Janer, Jordi},
+  journal={Journal of Electrical and Computer Engineering},
+  volume={2016},
+  year={2016},
+  publisher={Hindawi}
+}
+@article{patynen2008anechoic,
+  title={Anechoic recording system for symphony orchestra},
+  author={P{\"a}tynen, Jukka and Pulkki, Ville and Lokki, Tapio},
+  journal={Acta Acustica united with Acustica},
+  volume={94},
+  number={6},
+  pages={856--865},
+  year={2008},
+  publisher={S. Hirzel Verlag}
+}
+"""
+
+REMOTES = {
+    "all": download_utils.RemoteFileMetadata(
+        filename="PHENICX-Anechoic.zip",
+        url="https://zenodo.org/record/840025/files/PHENICX-Anechoic.zip?download=1",
+        checksum="7fec47568263476ecac0103aef608629",
+        # destination_dir="..",
+        unpack_directories=["PHENICX-Anechoic"],
+    )
+}
+
+LICENSE_INFO = """
+Creative Commons Attribution Non Commercial Share Alike 4.0 International
+"""
+
+DATASET_SECTIONS = {
+    "doublebass": "strings",
+    "cello": "strings",
+    "clarinet": "woodwinds",
+    "viola": "strings",
+    "violin": "strings",
+    "oboe": "woodwinds",
+    "flute": "woodwinds",
+    "trumpet": "brass",
+    "bassoon": "woodwinds",
+    "horn": "brass",
+}
+
+
+class Track(core.Track):
+    """Phenicx-Anechoic Track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        audio_path (list): path to the audio files
+        notes_path (list): path to the audio files
+        notes_original_path (list): path to the audio files
+        instrument (str): the name of the instrument
+        piece (str): the name of the piece
+        n_voices (int): the number of voices in this instrument
+        track_id (str): track id
+
+    Cached Properties:
+        melody (F0Data): melody annotation
+
+    """
+
+    def __init__(
+        self,
+        track_id,
+        data_home,
+        dataset_name,
+        index,
+        metadata,
+    ):
+        super().__init__(
+            track_id,
+            data_home,
+            dataset_name,
+            index,
+            metadata,
+        )
+
+        self.instrument = self.track_id.split("-")[1]
+        self.piece = self.track_id.split("-")[0]
+
+        self.audio_paths = [
+            os.path.join(self._data_home, self._track_paths[key][0])
+            for key in self._track_paths
+            if "audio_" in key
+        ]
+
+        self.n_voices = len(self.audio_paths)
+
+        self.notes_path = os.path.join(self._data_home, self._track_paths["notes"][0])
```

Comment:
remember to update these if we merge #468 first!

Reply:
@nkundiushuti Looks like we should indeed update these. I'm happy to commit to your branch with the fix if you're sick of this PR :)